tests/test_measure_script.py

```diff
@@ -0,0 +1,84 @@
+"""Tests for the measure script code."""
+import pytest
+
+from finesse.gui.measure_script.parse import ParseError, parse_script
+
+
+def test_valid_script() -> None:
```

Comment:
As pointed out in another PR, this could be made more concise using parametric testing.

Reply:
Good shout. I wrote this before that discussion, but you're quite right! I'll do that.